### DISCUSSION TITLE
How to properly make Linux Mint look like a Windows OS

### DISCUSSION THREAD

**Discussion by ghost** [linuxmint/discussions#606] [Questions]
I have tried b00merang's themes, but they don't look to apply well. Here's what I have got:

![image](https://github.com/user-attachments/assets/514404e4-19b1-4055-8f89-2dc63fe98d49)

Note:

- The Visual Studio Code icon has been manually put by me from a SVG download from a Google Images search.
- The Start icon should be white, not blue; but it's what was there.

![image](https://github.com/user-attachments/assets/c0bc3372-32fe-4b27-8b31-c9420e1d5e5a)

The start menu is not ugly, but it's not the most consistent as you can see.

I'd like either Vista, 8.1 or 10 (2015 build specifically).

**Comment by mtwebster** [comment]
This is better answered in the forums I think.

https://forums.linuxmint.com/
